Sources/SendGridKit/Models/EmailValidation.swift

```diff
@@ -0,0 +1,279 @@
+/// A request to validate an email address using SendGrid's Email Validation API.
+public struct EmailValidationRequest: Encodable {
+    /// The email address to validate.
+    public let email: String
+
+    /// Optional source of the email address being validated.
+    public let source: String?
+
+    /// Initialize a new `EmailValidationRequest`
+    ///
+    /// - Parameters:
+    ///   - email: The email address to validate
+    ///   - source: Optional source of the email address being validated
+    public init(email: String, source: String? = nil) {
+        self.email = email
+        self.source = source
+    }
+}
+
+/// A request to get an upload URL for bulk email validation.
+public struct BulkValidationUploadURLRequest: Encodable {
+    /// The file type that will be uploaded.
+    public let fileType: FileType
+
+    /// Initialize a new `BulkValidationUploadURLRequest`
+    ///
+    /// - Parameter fileType: The file type that will be uploaded (CSV or ZIP)
+    public init(fileType: FileType) {
+        self.fileType = fileType
+    }
+
+    /// The type of file that will be uploaded for bulk validation.
+    public enum FileType: String, Codable, Sendable {
+        /// A CSV file containing email addresses.
+        case csv = "csv"
+        /// A ZIP file containing a CSV file with email addresses.
+        case zip = "zip"
```

Comment:
If the `case` name is the same as the underlying string you don't need to specify it
```suggestion
        case csv
        /// A ZIP file containing a CSV file with email addresses.
        case zip
```